src/constants.js

```diff
@@ -9,7 +9,7 @@ var __assign = (this && this.__assign) || Object.assign || function(t) {
     }
```

Comment:
As the amount of constants growing fast, constants should be well organized.

Reply:
After refactor with SDK, the constants will be less, because a lot of constants are packaged in SDK.